pkgs/by-name/ca/caido/package.nix

```diff
@@ -21,8 +21,6 @@ in appimageTools.wrapType2 {
   extraInstallCommands = ''
     mv $out/bin/${pname}-${version} $out/bin/${pname}
     install -m 444 -D ${appimageContents}/caido.desktop -t $out/share/applications
-    substituteInPlace $out/share/applications/caido.desktop \
-      --replace 'Exec=AppRun' 'Exec=${pname}'
```

Comment:
Why this has been removed?

Reply:
```
[Desktop Entry]
Categories=Development;
Comment=The lightweight web security auditing toolkit
Exec=caido
Icon=caido
Name=Caido
Terminal=false
Type=Application
```

The new .AppImage doesn't contain Exec=AppRun, so I removed it.